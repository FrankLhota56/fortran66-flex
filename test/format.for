*
* FORMAT test
*
101   Format (/-1P2D4.3, 3X// / A4, 5HHello/)
222   format (- 1 p  2 d 4 . 3)
      End
