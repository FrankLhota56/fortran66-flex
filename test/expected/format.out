Line     4: INT_LIT(101)
Line     4: FORMAT
Line     4: '('
Line     4: FIELD_SEP
Line     4: FIELD_DESC(-1P2D4.3)
Line     4: ','
Line     4: FIELD_DESC(3X)
Line     4: FIELD_SEP
Line     4: FIELD_DESC(1A4)
Line     4: ','
Line     4: HOLLERITH_LIT(Hello)
Line     4: FIELD_SEP
Line     4: ')'
Line     5: EOS
Line     5: INT_LIT(222)
Line     5: FORMAT
Line     5: '('
Line     5: FIELD_DESC(-1P2D4.3)
Line     5: ')'
Line     6: EOS
Line     6: END
